spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/AvroBinaryIterator.java

```
@@ -73,6 +82,11 @@ public boolean hasNext() {
   @Override
   public InternalRow next() {
     try {
+      // TODO(nileshny) : confirm if this is correct way to get number of Avro rows parsed
+      try {
+        numberOfRowsParsed = Math.addExact(numberOfRowsParsed, 1);
+      } catch (ArithmeticException ae) {;
```

Empty catch - either throw or explain why it is ignored